### DISCUSSION TITLE
Approved co-pilot but no access

### DISCUSSION THREAD

**Discussion by TheGuyuKnow** [community/community#110226] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I recieved the approved notification the other day
<img width="241" alt="bild" src="https://github.com/community/community/assets/161236090/8217d5e2-8d0f-4656-afbb-e2582da4d446">
and as Im in a hurry with a project Im doing, I can't stop but wonder when I can get access since both "your copilot" and vissual studio tell me I don't have it. (Ive tried logging out & signing in) 
If all I need to do is wait then thats that, but if not, any ideas?

**Comment by grandmas-favorite** [comment]
Hi @TheGuyuKnow!

You are more likely to get a useful response if you are posting your question in the applicable category, the Actions category is solely related to conversations around the GitHub Actions tool. I've gone ahead and moved it for you. Good luck!
